executor/aggfuncs/aggfuncs.go

```diff
@@ -151,8 +151,8 @@ type baseAggFunc struct {
 	ordinal int
 }
 
-func (*baseAggFunc) MergePartialResult(sctx sessionctx.Context, src, dst PartialResult) error {
-	return nil
+func (*baseAggFunc) MergePartialResult(sctx sessionctx.Context, src, dst PartialResult) (int64, error) {
+	return int64(0), nil
```

Comment:
How about removing `int64` here since `return 0, nil` is OK?

Reply:
done